UPGRADE-4.2.md

```diff
@@ -125,6 +125,26 @@ HttpKernel
 FrameworkBundle
 ---------------
 
+ * The `allow_no_handler` middleware has been removed. Use `framework.messenger.[bus].default_middleware` instead:
+
+   Before:
+   ```yaml
+   framework:
+       messenger:
+           events:
+               middleware:
+                 - allow_no_handler
+   ```
+
+   After:
+   ```yaml
+   framework:
+       messenger:
+           events:
+               default_middleware: allow_no_handlers
+   ```
+
+ * The `messenger:consume-messages` command expects a mandatory `--bus` option value if you have more than one bus configured.
```

Comment:
doesn't it consume message for all buses when none is provided?

Reply:
Nope. The `--bus` option is about where the received message is dispatched back. It's independent from where was initially dispatched the message.
If none is provided, interactively, we'll ask to select one.